src/main/target/SITL/target.c

```diff
@@ -505,11 +592,17 @@ static void pwmCompleteMotorUpdate(void)
     if (pthread_mutex_trylock(&updateLock) != 0) return;
     udpSend(&pwmLink, &pwmPkt, sizeof(servo_packet));
 //    printf("[pwm]%u:%u,%u,%u,%u\n", idlePulse, motorsPwm[0], motorsPwm[1], motorsPwm[2], motorsPwm[3]);
+    udpSend(&pwmRawLink, &pwmRawPkt, sizeof(servo_packet_raw));
 }
 
 void pwmWriteServo(uint8_t index, float value)
 {
     servosPwm[index] = value;
+    if (index + pwmRawPkt.motorCount < SIMULATOR_MAX_PWM_CHANNELS)
+    {
```

Comment:
```suggestion
    if (index + pwmRawPkt.motorCount < SIMULATOR_MAX_PWM_CHANNELS) {
```
Please check: https://betaflight.com/docs/development/CodingStyle